localization/pose_estimator/ndt_scan_matcher/ndt_omp/package.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0"?>
-<package>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?> 
```

Comment:
I have seen and preserved 
```

<?xml version="1.0"?>
<package format="2">
```
elsewhere. Are you following the recommendations and I don't? What are the implications?

Reply:
I have seen both package format 2 & 3 as well. 
From here 
https://www.ros.org/reps/rep-0149.html
format 3 seems to enable mixing ros1 ad ros2 builds.

Reply:
The migration guide says "ROS 2 doesn’t support format 1 of the package specification but only newer format versions (2 and higher)." My understanding is that if you have a package that's already at version 2 it's fine to leave it, but this one was at version 1, and going straight to 3 doesn't hurt.